src/pudl/validate.py

```diff
@@ -48,16 +47,16 @@ def no_null_rows(
         ``max_null_fraction``.
     """
     if cols == "all":
-        cols = df.columns
+        cols = list(df.columns)
 
     null_rows = df[cols].isna().sum(axis="columns") / len(cols) > max_null_fraction
     if null_rows.any():
         raise ExcessiveNullRowsError(
             message=(
-                f"Found {null_rows.sum(axis='rows')} excessively null rows in {df_name}.\n"
+                f"Found {len(null_rows)} excessively null rows in {df_name}.\n"
```

Comment:
It looks like this change switches from printing out the number of rows failing our excessively-null test, to printing out the total number of rows, null or not. Check my read?

```
df: r rows, c columns
df[cols]: r rows, |cols| columns
df[cols].isna(): r rows, |cols| columns, 1 if na, 0 otherwise
df[cols].isna().sum(axis="columns"): r rows, 1 column, number of null columns in each row
df[cols].isna().sum(axis="columns") / len(cols): r rows, 1 column, percentage of null columns in each row
df[cols].isna().sum(axis="columns") / len(cols) > max_null_fraction: r rows, 1 column, 1 if excessively null, 0 otherwise
```

```suggestion
                f"Found {null_rows.sum()} excessively null rows in {df_name}.\n"
```